### PR TITLE
add unfriending, liking comments, clickable mini-profiles, frontend tests

### PR DESCRIPTION
**Pull request by seliayeu**

### CODE REVIEW

File: src/components/AuthorPreview.jsx [hunk 1]
```diff
@@ -14,14 +14,15 @@ const AuthorPreview = ({ authorData }) => {
   }
 
   const parseId = (fullId) => {
+    console.log(["epic", "cool", "sick"].at(-1))
```
Comment (Xiaoyu-Ben-Wang) [inline src/components/AuthorPreview.jsx:17]: this line shouldn't be here

File: src/components/Comment.jsx [hunk 2]
```diff
@@ -1,11 +1,43 @@
...
   const CMParser = new Parser({ safe: true });
   const CMWriter = new HtmlRenderer();
 
-  console.log(comment)
+  useEffect(() => {
+    console.log(comment)
+    if (comment?.id === undefined) return;
+    const getCommentLikes = async () => {
+      const res = await postService.getCommentLikes(comment.id.split("/").at(-5), comment.id.split("/").at(-3), comment.id.split("/").at(-1));
+      console.log(res);
```
Comment (Xiaoyu-Ben-Wang) [inline src/components/Comment.jsx:21]: remove console.log()

File: src/components/AuthorPreview.jsx (hunk 1 above)
Comment (seliayeu) [reply src/components/AuthorPreview.jsx:17]: excellent point team mate. i will make quick work of this!

File: src/components/Comment.jsx (hunk 2 above)
Comment (seliayeu) [reply src/components/Comment.jsx:21]: well caught team mate. i will get to this at once.

File: src/pages/Home/index.jsx [hunk 3]
```diff
@@ -53,11 +53,14 @@ const Home = ({ inbox, setInbox, followers }) => {
     }
   };
 
+  console.log(inbox);
```
Comment (Xiaoyu-Ben-Wang) [inline src/pages/Home/index.jsx:56]: console.log() here

File: src/services/post.js [hunk 4]
```diff
@@ -34,26 +26,13 @@ const likePost = async (csrfToken, authorId, {author, object}) => {
...
-
-
-
 // creates post by author with
 //    contentType: {text/markdown, text/plain, application/base64, image/png;base64, image/jpeg;base64}
 //    content
 //    title
 //    categories
 const createPost = async (csrfToken, authorId, postData) => {
+  console.log(postData)
```
Comment (Xiaoyu-Ben-Wang) [inline src/services/post.js:35]: console.log() here

**Review by Xiaoyu-Ben-Wang — CHANGES_REQUESTED** [review]

**Review by Xiaoyu-Ben-Wang — APPROVED** [review]

**Pull request merged**